shopping-cart/shoppingcart.proto

```diff
@@ -38,25 +37,14 @@ message Cart {
 
 service ShoppingCart {
     rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
-        option (google.api.http) = {
-            post: "/com.example.shoppingcart.ShoppingCart/cart/{user_id}/items/add",
-            body: "*",
-        };
-        option (.cloudstate.eventing).in = "items";
+        option (google.api.http).post = "/com.example.shoppingcart.ShoppingCart/cart/{user_id}/items/add";
```

Comment:
I'm not familiar with why we no longer need the body option...

Reply:
I wonder why did it was needed in the first place. Anyway, everything seems to work fine without it.

Reply:
@justinhj, `body: "*"` is needed for HTTP transcoding. However, HTTP is not used at the moment. Anyway, I returned it back.